htmlgen.py

```diff
@@ -245,7 +245,7 @@ def pretty_dur(dur):
   try:
     secs = dur % 60
   except:
-    print("FAIL on %s" % dur)
```

Comment:
I renamed this file to prevent issues with importing `mako.html` which popped up

Reply:
Looks good